### ISSUE TITLE
cant break blocks

### ISSUE THREAD

**Issue by sengia2002** [davqvist/Exoria#346]
when ever i try to break a block it disappears and when you walk over it you glitch in it and you stuck.
if i right click the broken blocks it just comes back

**Comment by davqvist** [comment]
lol what? I have never heard of that. I need more info. What version of the pack? Did it happen from the start or just now? Does it happen in a new world? SIngleplayer or Multiplayer? Which tool are you using? Does it happen with other tools? etc.

**Comment by sengia2002** [comment]
i use the most recent update and yes i made like 5 world already  every single 1 does it. tool no tooll doesn't make a diffrence even if i go in to gamemode 1. i play on single player

**Comment by sengia2002** [comment]
it did start ever since i downloaded it i almost feel like its lag but my fps etc. is fine so not sure also i tried to go away from the spawn chunk but still te same

**Comment by zZSandManZz** [comment]
I decided to try this pack after seeing a few YouTube videos about it. first world didn't have a portal neither did the second one but the 3rd one did. and then I though finally i can play. First experience was I cant break shite

then i gave up and started looking around for an answer and well the only thing I've found so far is this thread and it was absolutely no help

**Comment by davqvist** [comment]
Like I said above no idea. Maybe you can give relevant info. What version of the pack? Did it happen from the start or just now? Does it happen in a new world? SIngleplayer or Multiplayer? Which tool are you using? Does it happen with other tools? etc.

**Comment by zZSandManZz** [comment]
it is the latest version of the pack(on the twitch launcher at least 1.3.0c), single,player, its been happening since the start. I have made 7 worlds and each one of them have the same issue i reinstalled the pack and everything

**Comment by zZSandManZz** [comment]
the way i can explain it is kinda like if you ever played Vanilla MC and you have a haste 2 beacon and a eff5 pick on a multiplayer server and some times you get the invisible blocks you glitch into. Its like that but EVERY block does it if you break it

**Comment by sengia2002** [comment]
yup and no matter if you wait for an hour or relog the blocks just reapear like you didnt even thouched them

**Comment by chubbytoad** [comment]
I am having the same issue, 1.3.0c, installed yesterday from curseforge using GDLauncher. I tried Vanilla Fix, turning off particles, lowering render distance, etc.

There is a block desync(?); visually the block is broken, but there is no drop and a phantom block persists and right clicking makes the block reappear.

Necro, I know

Edit: updated Gamestages and FoamFix to the latest versions and it appears to be working
